src/config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  apiUrl: 'http://3.73.84.195:49153'
```

Comment:
how you are planning to handle different urls for different env's (local, test, stage, prod)?